src/constants/volumeRayMarchShader.ts

```diff
@@ -184,7 +206,7 @@ vec4 integrateVolume(vec4 eye_o,vec4 eye_d,
   float invstep = (tfar-tnear)/csteps;
   // special-casing the single slice to remove the random ray dither.
   // this removes a Moire pattern visible in single slice images, which we want to view as 2D images as best we can.
-  float r = (SLICES==1.0) ?  0.0 : 0.5 - 1.0*rand(eye_d.xy);
+  float r = (SLICES==1.0) ? 0.0 : rand(eye_d.xy);
```

Comment:
so the difference here is that the ray offsets are always pushed in to the volume edge instead of distributed half in and half out.  theoretically they are scaled down enough that they shouldn't substantially affect the overall brightness at the edges but just want to be aware that that is what's changing here.

Reply:
There is an effect in no-interpolation mode where some voxels appear grainy when the volume is viewed edge-on, as ray steps per voxel becomes very small and some rays end up missing some voxels entirely (I tried to capture this a bit in the second screenshot above). I didn't investigate to my satisfaction if that was improved when this offset was shifted back by a half, but I did try shrinking the offset by some constant factor, e.g. `rand(eye_d.xy)*0.5`, and saw the moire pattern reappear pretty quickly.